### PR TITLE
Upgrade instructions to OCP4.9 / Administrator Acknowledgment

### PR DESCRIPTION
**Pull request by bastjan**
This page shows steps to:
- Find services using deprecated APIs.
- Provide the required acknowledgment.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/update_maintenance/v_4_9.adoc [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+. Check requests to removed APIs
++
+[source,bash]
+----
+kubectl get apirequestcounts \
+  --as=cluster-admin \
+  -o jsonpath='{range .items[?(@.status.removedInRelease!="")]}{.status.removedInRelease}{"\t"}{.metadata.name}{"\t"}{.status.requestCount}{"\n"}{end}'
+----
++
+Should print something like the sample below.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/update_maintenance/v_4_9.adoc:19]:
```suggestion
Should print a report like the sample below.
```

**Review by ccremer — COMMENTED** [review]
1) Please clarify the title so it mentions the approval
2) Is this actively decided that this is the way to go, resp. did we decide against managing the approval in code?

**Comment by bastjan** [comment]
> did we decide against managing the approval in code?

I remember we did it in the backlog refinement. Sadly we did not write it down.

File: docs/modules/ROOT/pages/how-tos/update_maintenance/v_4_9.adoc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+----
+kubectl get apirequestcounts customresourcedefinitions.v1beta1.apiextensions.k8s.io \
+  --as=cluster-admin \
+  -o jsonpath='{range ..username}{$}{"\n"}{end}' \
+  | sort | uniq
+----
++
+[NOTE]
+Some services autodiscover supported APIs, notably https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/[kube-controller-manager] (`system:kube-controller-manager`) and ArgoCD (`system:serviceaccount:syn:argocd-application-controller`).
+These services will show up as using deprecated APIs.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/update_maintenance/v_4_9.adoc:41]: just to be consistent with the important note in the other page
```suggestion
[NOTE]
====
Some services autodiscover supported APIs, notably https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/[kube-controller-manager] (`system:kube-controller-manager`) and ArgoCD (`system:serviceaccount:syn:argocd-application-controller`).
These services will show up as using deprecated APIs.
====
```

**Review by ccremer — APPROVED** [review]

**Review by simu — APPROVED** [review]

**Pull request merged**